subsys/bluetooth/mesh/light_ctrl_srv.c

```diff
@@ -1320,13 +1329,18 @@ static int light_ctrl_srv_start(struct bt_mesh_model *mod)
 		store(srv, FLAG_STORE_STATE);
 		break;
 	case BT_MESH_ON_POWER_UP_ON:
-		ctrl_disable(srv);
+		if (atomic_test_bit(&srv->flags,
+		FLAG_CTRL_SRV_MANUALLY_ENABLED)) {
+			ctrl_enable(srv);
+			light_set(srv,
+				(srv->lightness->default_light ?
+					srv->lightness->default_light :
+					srv->lightness->last),
+				srv->lightness->ponoff.dtt.transition_time);
```

Comment:
`light_set` should be called in the else-branch to preserve the old behavior when the server hasn't been manually enabled.

Reply:
Done